final/design/eval-strat.tex

```diff
@@ -0,0 +1,38 @@
+\section{Evaluation Strategies}
+\label{sec:eval-strat}
+As explained in \cref{sec:supp-diff-ways}, there are multiple ways for
+specifying the dynamic semantics of a language. Interpretation can for example
+be performed through an interpreter generated from a DynSem specification, or by
+implementing the interpreter in Stratego or Java. However, in the end all of
+these methods do the same thing, that is they accept an AST as input and somehow
+transform it into an output. In Spoofax, both the input and the output can be
+represented as a Stratego term. It is only the internal representation of the
```

Comment:
Perhaps this requires an explanation? Not sure if this can be done in one sentence, but otherwise you can perhaps refer to documentation in a footnote or something.